fabric8-maven-plugin/src/main/java/io/fabric8/maven/JsonMojo.java

```diff
@@ -1157,7 +1163,7 @@ protected boolean addServiceAccountIfIUsingSecretAnnotations(KubernetesListBuild
             secrets.add(secretRef);
         }
 
-        if (!secrets.isEmpty()) {
+        if (!secrets.isEmpty() || createServiceAccount) {
```

Comment:
good point; though quite a few apps reuse the `fabric8` SA so we do wanna be able to disable it. 

Maybe we enable it by default? (then find all uses of fabric8 SA and disable creation on everything but the console)